deployment/debug/debug-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
```

Comment:
I'm not convinced that a service is appropriate here.  I don't know the envoy debug interface that well though.

If I'm running 10 replicas of Contour this will randomly hit one of them at each web request.  That might not be what users expect.

We could also go with `service.spec.sessionAffinity` to create some predictability but that'll change if the set of pods changes.

(What I really want to do is find time to finish up my imaginary super version of `kubectl port-forward` as that would make it much easier to debug this stuff.)

Reply:
iff we trust that the contour / envoy connection is working then each envoy will get the same set of listeners / routes / clusters, etc.

This is pretty low level stuff, but at the moment I don't have a way of helping users understand what envoy is doing when they introduce ingress x, #188 for example. I know ('cos I wrote this junk) what is going on, but saying "trust me" isn't as useful as showing users how to read the configuration envoy has applied.